dash_bio/component_factory/_clustergram.py

```diff
@@ -61,77 +61,85 @@ def Clustergram(
-    :param (list) colorMap: Optional colorscale for dendrogram tree
-    :param (dict) colorList: Optional list of colors to use for row and col
+    :param (list) color_map: Optional colorscale for dendrogram tree
+    :param (dict) color_list: Optional list of colors to use for row and col
                              dendrograms
-    :param (double) displayRange: Standardized values from the dataset that
+    :param (double) display_range: Standardized values from the dataset that
                                   are below the negative of this value will
                                   be colored with one shade, and the values
                                   that are above this value will be colored
                                   with another in the heatmap (3 by default)
-    :param (bool) symmetricValue: Forces the colorscale of the heatmap to be
+    :param (bool) symmetric_value: Forces the colorscale of the heatmap to be
                                   centered around zero (True by default)
-    :param (bool) logTransform: Transforms the data to a logarithmic axis
+    :param (bool) log_transform: Transforms the data to a logarithmic axis
                                 with a basis of two (False by default)
-    :param (list/float) displayRatio: The dendrograms' heights with respect to
+    :param (list/float) display_ratio: The dendrograms' heights with respect to
                                      the size of the heatmap; with one element,
                                      both the row and column dendrograms have
                                      the same ratio; with two, the row
                                      dendrogram ratio corresponds to the first
                                      element of the list (default: 0.2)
-    :param (function) imputeFunction: The function used to impute missing data.
-                                      It should take as input the dataset and
-                                      output a dataset with imputed values.
-    :param (list[dict]) rowGroupMarker: Specifies which rows to annotate; each
+    :param (dict) imputer_parameters: Specifies the parameters 'missing_values'
+                                      and 'strategy' of the SimpleImputer class
+                                      from scikit-learn 0.20.3. An additional
+                                      parameter, 'axis', is used to specify the
+                                      direction along which to impute (a parameter of
+                                      Imputer, which was deprecated in scikit-learn 0.20.0).
+                                      'axis=0' indicates that imputing should happen along columns,
+                                      while 'axis=1' indicates that it should happen
+                                      along rows (see:
+                                      http://lijiancheng0614.github.io/scikit-learn/modules/
```

Comment:
Wait, why this personal webpage and not the official scikit-learn docs?

Reply:
Oops -- copy/pasted the  wrong link.